README.md

```diff
@@ -11,58 +11,21 @@ Use of NVIDIA cuOpt is subject to the [End User License Agreement](https://docs.
 
 ## [NVIDIA cuOpt](https://developer.nvidia.com/cuopt-logistics-optimization)
 
-cuOpt uses highly optimized GPU-accelerated solvers relying on heuristics, metaheuristics, and optimization. In addition to providing dramatically accelerated, world class solutions to some of the most difficult optimization problems, NVIDIA cuOpt prioritizes ease of use through high level Python and REST based microservice APIs
+cuOpt uses highly optimized GPU-accelerated solvers relying on heuristics, metaheuristics, and optimization. In addition to providing dramatically accelerated, world class solutions to some of the most difficult optimization problems, NVIDIA cuOpt prioritizes ease of use through high level Python SDK and service APIs
```

Comment:
Should we remove sdk part ?

Reply:
As per the current plan, there will be a path to the SDK.  I think we should leave it.